frontend/src/components/indexPage/FilterSection.js

```diff
@@ -82,49 +62,18 @@ export default function FilterSection({
     setFiltersExpanded(!filtersExpanded);
   };
 
-  const onSearchValueChange = (type, newValue) => {
-    setSearchFilters({ ...searchFilters, [type]: newValue });
-  };
-
-  const onSearchSubmit = async (type) => {
-    const newUrlEnding = buildUrlEndingFromSearch(searchFilters[type]);
-    if (state.urlEnding[type] != newUrlEnding) {
-      try {
-        let filteredItemsObject;
-        if (type === "projects") filteredItemsObject = await getProjects(1, token, newUrlEnding);
-        else if (type === "organizations")
-          filteredItemsObject = await getOrganizations(1, token, newUrlEnding);
-        else if (type === "members") {
-          filteredItemsObject = await getMembers(1, token, newUrlEnding);
-          filteredItemsObject.members = membersWithAdditionalInfo(filteredItemsObject.members);
-        } else {
-          console.log("cannot find type!");
-        }
-        setState({
-          ...state,
-          items: { ...state.items, [type]: filteredItemsObject[type] },
-          hasMore: { ...state.hasMore, [type]: filteredItemsObject.hasMore },
-          urlEnding: { ...state.urlEnding, [type]: newUrlEnding },
-          nextPages: { ...state.nextPages, [type]: 2 },
-        });
-      } catch (e) {
-        console.log(e);
-      }
-    }
-  };
```

Comment:
Hoist all search submit handling to `browse`, and dedupe state management there. This should simplify how we're handling state management with search, and loading.